samples/nextjs/sitecore/definitions/components/PersonalizationLoadingComponent.sitecore.ts

```diff
@@ -0,0 +1,20 @@
+// eslint-disable-next-line no-unused-vars
```

Comment:
What variable is unused on the next line?

Reply:
removed